### ISSUE TITLE
Create a file when not empty but dont match anything

### ISSUE THREAD

**Issue by otavioschwanck** [nvim-telescope/telescope-file-browser.nvim#120]
Continuation of: https://github.com/nvim-telescope/telescope-file-browser.nvim/issues/114

Sometimes, i want to create something that matches some file, like:

![image](https://user-images.githubusercontent.com/9551316/161595058-84fe905c-758b-458f-b950-3cb28c986b63.png)

(want to create auto.vim but already have autopairs.vim).

Would be nice to have an item named [+] auto.vim that when i select, it creates the file.

**Comment by fdschmidt93** [comment]
We already have two very consistent ways of creating files now after #114.

I'm inclined to accept a PR, but it won't work that way because it's not how telescope works. Also, I'm not sure how this is supposed to work really, because it's unclear when or when not to sure the extra item.

I'd suggest getting a deeper understanding of telescope and https://github.com/nvim-telescope/telescope-file-browser.nvim/blob/c6f5104ff309649ebbaec283bbd1ab54511dd109/lua/telescope/_extensions/file_browser.lua#L96-L134 providing your own action select replace logic.

We can also augment the PR that closed #114 and create a file by default on Shift-Enter or something instead of when results are only empty. Probably better approach. I'd very much welcome a PR as I don't really need the feature (though it's cool to have) and other than fixing obvious bugs, I prefer spending my time otherwise.

**Comment by otavioschwanck** [comment]
> 

I loved the idea of shift + enter.  Maybe can be default?

**Comment by fdschmidt93** [comment]
As mentioned, PR welcome.

**Comment by otavioschwanck** [comment]
> As mentioned, PR welcome.

i dont have any experience with lua or telescope.  Any tips of how to implement this?

I Tried create a action that calls make_entry, but i dont know how to get the args

**Comment by fdschmidt93** [comment]
Look at

- `pickers.lua` in telescope -- there's how you can `get_prompt` (it's literally called something like that)
- `actions/init.lua` in telescope -- github create branch action also infers new branch name from prompt
- `actions.lua` `fb_actions.create` from `telescope-file-browser.nvim` how to make create a file action with plenary

Then you'd map that action to "<Shift-Enter>". This might require 0.7 as there were recent patches to distinguish key codes and stuff.

**Comment by fdschmidt93** [comment]
Closed via #130
